bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -405,6 +412,11 @@ def GetCanvasDrawers():
     lOfC = ROOT.gROOT.GetListOfCanvases()
     return [NotebookDrawer(can) for can in lOfC if can.IsDrawn()]
 
+def GetRCanvasDrawers():
+    if not RCanvasAvailable(): return []
+    lOfC = ROOT.Experimental.RCanvas.GetCanvases()
+    return [NotebookDrawer(can.__smartptr__().get()) for can in lOfC if can.IsShown()]
```

Comment:
`can.__smartptr__().get()` should not be necessary as `can` should already behave as an RCanvas, can you try just constructing with `NotebookDrawer(can)`?

Reply:
I tried - it does not work. Copy of `shared_ptr` does not work properly.